docs/08-developer-guide/01-build/02-maven.md

```diff
@@ -8,3 +8,99 @@ sidebar_label: Maven
 
 - Cover basic Maven commands to build and run tests.
 - Document all the Ozone specific Maven flags we use to speed up or skip parts of the build, and when they are useful.
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This guide explains how to build Apache Ozone from source using Maven and prepare it for deployment.
+
+## Prerequisites
+
+**TODO** : [HDDS-11625](https://issues.apache.org/jira/browse/HDDS-11625) Finalize the version numbers of prerequisite packages
+
+Before you begin, ensure you have the following installed on your build machine:
+
+- Java 1.8 or higher
+- Apache Maven 3.6.3 or higher
+- Git (if building from source repository)
+
+## Building Ozone
+
+You can build Apache Ozone either by cloning the source code from Git or by downloading the official source tarball.
+
+### 1. Obtain the Source Code
+
+Choose one of the following methods to get the source code:
+
+<Tabs>
+  <TabItem value="Git" label="Git" default>
+    ```bash
+    git clone https://github.com/apache/ozone.git
+    cd ozone
+    ```
+  </TabItem>
+  <TabItem value="Tarball" label="Tarball">
```

Comment:
>We should also clarify that the published tarballs correspond to official releases only. We don't publish snapshots so if an intermediate build is desired it will have to be done from source.

I am not sure I follow what needs to be clarified. This page is about building from source. The official sources will have the source code corresponding to the build and the git option is an alternative to get the other versions of the source code.

Reply:
Fixed.